caom2-search-server/src/main/resources/META-INF/resources/js/cadc.search.app.js

```diff
@@ -1316,6 +1316,12 @@
                     qKey,
                     decodeURIComponent(qValue.join())
                   )
+                } else if (qKey === 'Plane.time.bounds.samples_PRESET') {
+                  activeSearchForm.setSelectValue(
+                    ca.nrc.cadc.search.TIME_BOUNDS_FIELD_ID,
+                    qKey,
+                    decodeURIComponent(qValue.join())
+                  )
```

Comment:
This won't work for the ObsCore search form.  It needs to handle the `t_coverage` field.